src/plugins/intel_cpu/src/nodes/kernels/x64/brgemm_kernel.cpp

```diff
@@ -134,9 +140,14 @@ void BrgemmKernel::init_brgemm(brgemmCtx& ctx,
 
     const bool is_int8 =
         one_of(ctx.dt_in0, data_type::u8, data_type::s8) && one_of(ctx.dt_in1, data_type::u8, data_type::s8);
-    auto isa = use_amx                                     ? isa_undef
-               : ctx.dt_in0 == dnnl_data_type_t::dnnl_bf16 ? avx512_core_bf16
-                                                           : (is_int8 ? avx512_core_vnni : avx512_core);
+    cpu_isa_t isa;
+    if (mayiuse(avx512_core)) {
+        isa = use_amx ? isa_undef
+                : ctx.dt_in0 == dnnl_data_type_t::dnnl_bf16 ? avx512_core_bf16
+                                                            : (is_int8 ? avx512_core_vnni : avx512_core);
+    } else {
+        isa = cpu_isa_t::avx2;
```

Comment:
Is AVX2 a must need for PA ? When will AVX2 be selected ?

Reply:
AVX2 may use for testing. Not sure real case needs it.